plugins/base_commands.py

```diff
@@ -8,8 +8,12 @@
 
 import datetime
 
+import logging
 from spockbot.plugins.base import PluginBase
 from spockbot.vector import Vector3
+import json
+tpareqs = {}
```

Comment:
Why is this a global. Like, I'm all for throwing coding practice to the wind, just ask @txomon or @gamingrobot. But for next time, random module scoped globals aren't a super great thing for me to see in a PR.